### PR TITLE
fix: language-server v4.12.0 @W-16395004@

### PR DESCRIPTION
**Pull request by ekashida**
### What does this PR do?

Bumps the version of @salesforce/lightning-lsp-common, @salesforce/aura-language-server, and @salesforce/lwc-language-server to add the ability for the language server to update the paths mapping in tsconfig.sfdx.json

https://github.com/forcedotcom/lightning-language-server/pull/594

@W-16395004@

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -7,4 +7,6 @@ save-exact=true
 engine-strict=true
 
 # Smaller on disk than lockfile version 2, but not interoperable with older npm versions. Ideal if all users are on npm version 7 and higher.
-lockfile-version=3
\ No newline at end of file
+lockfile-version=3
+
+registry=https://registry.npmjs.org
```
Comment (peternhale) [inline .npmrc:12]: Please revert the change to `.npmrc`

**Review by peternhale — CHANGES_REQUESTED** [review]

**Comment by ekashida** [comment]
Thanks for the review @peternhale. It looks like some of the checks are stalled, any idea what may be going on?

**Comment by CristiCanizales** [comment]
@ekashida I just kicked the runs with an empty push🙂

**Review by peternhale — APPROVED** [review]

**Comment by peternhale** [comment]
@ekashida you will need to update your branch with HEAD of develop before we can merge the PR

**Comment by ekashida** [comment]
@peternhale looks like it's ready to merge!

**Pull request merged**